src/test/run-pass/cfg/cfg-family.rs

```diff
@@ -10,3 +8,7 @@ pub fn main() {
 #[cfg(unix)]
 pub fn main() {
 }
+
+#[cfg(not(any(windows, unix)))]
```

Comment:
I still this [@rasendubi was correct about this change defeating the point of the test](https://github.com/rust-lang/rust/pull/60386/commits/48f5b70e9055540622972cee05596ea60e0e9176#r279743858); I don't see how it could fail under this setup. Therefore, I'm not inclined to merge until that is addressed in some way. But the rest seems great!